lib/rack/utils.rb

```diff
@@ -248,13 +248,20 @@ def add_cookie_to_header(header, key, value)
           rfc2822(value[:expires].clone.gmtime) if value[:expires]
         secure = "; secure"  if value[:secure]
         httponly = "; HttpOnly" if (value.key?(:httponly) ? value[:httponly] : value[:http_only])
-        first_party = "; First-Party" if value[:first_party]
+        same_site = case (value[:first_party] || value[:same_site]).to_s.downcase
+        when "lax"
+          "; SameSite=Lax"
+        when "strict"
+          "; SameSite=Strict"
+        when "true"
+          "; SameSite"
```

Comment:
There were tests for "falsey" values that would break. I think it's simpler to just ignore any values other than `true` or case-insensitive "lax", or "strict."